pkg/processor/runtime/rpc/connection/abstract.go

```diff
@@ -470,13 +540,24 @@ func (be *AbstractEventConnection) SupportsRestart() bool {
 }
 
 func (be *AbstractEventConnection) Terminate() error {
-	return nuclio.ErrNotImplemented
+	// same as stop
+	return be.Stop()
 }
```

Comment:
In what flow is this `Terminate` called, and not the `Worker` terminate that signals the runtime?

Reply:
@TomerShor when allocator stops